sky/clouds/nebius.py

```diff
@@ -278,6 +296,28 @@ def check_credentials(cls) -> Tuple[bool, Optional[str]]:
                 f'{tenant_msg}')
         return True, None
 
+    @classmethod
+    def check_storage_credentials(cls) -> Tuple[bool, Optional[str]]:
```

Comment:
```suggestion
    def _check_storage_credentials(cls) -> Tuple[bool, Optional[str]]:
```
as a result of https://github.com/skypilot-org/skypilot/pull/4993 being merged